src/fontra/views/editor/panel-text-entry.js

```diff
@@ -43,11 +98,13 @@ export class SidebarTextEntry {
     );
   }
 
-  focusTextEntry() {
-    this.textEntryElement.focus();
+  fixTextEntryHeight() {
+    // This adapts the text entry height to its content
+    this.textEntryElement.style.height = "auto";
+    this.textEntryElement.style.height = this.textEntryElement.scrollHeight + 14 + "px";
   }
 
-  setupIntersectionObserver() {
+  setupInteractionObserver() {
```

Comment:
Please revert this rename, the original name is as intended.

Reply:
this too